pages/docs/visualizing-data/chart-menu.mdx

```diff
@@ -6,21 +6,21 @@ The more menu provides several actions related to the main chart. To access it c
 
 Downloading chart data allows you to download the aggregated data from a chart to a spreadsheet so you can export the data and use it in different applications. To do this:
 
-1. Select the Chart Menu on the top right of the main chart
-2. Select Download Data option
+1. Click the Chart Menu on the top right of the main chart.
+2. Click the `Download Chart Data` option.
 
 ## View Underlying Data
```

Comment:
I guess we're not sentence casing this yet?

Reply:
Not sentence casing this because it matches the uppercasing of the button that this references in the app.